resources/backupTemplates/pystein/templates/templates.json

```diff
@@ -0,0 +1,4968 @@
+        "host.json": "{\r\n    \"version\": \"2.0\",\r\n    \"logging\": {\r\n      \"applicationInsights\": {\r\n        \"samplingSettings\": {\r\n          \"isEnabled\": true,\r\n          \"excludedTypes\": \"Request\"\r\n        }\r\n      }\r\n    },\r\n    \"extensionBundle\": {\r\n      \"id\": \"Microsoft.Azure.Functions.ExtensionBundle\",\r\n      \"version\": \"[3.*, 4.0.0)\"\r\n    }\r\n  }",
```

Comment:
Alex mentioned that the `host.json` can't contain the `extensionBundle` property in order to debug (and you have denoted as such in the code.) I noticed that all the templates have host.jsons that include it by default, so whenever you create a new trigger, it automatically updates the `host.json` to include the property again.

Reply:
Ah, that's where the bundle was sneaking back in; we saw that yesterday but I hadn't yet pinpointed when it happened.  Agree that it could be removed from the (one) PyStein project template; I'll note that to the Functions team.  That said, the resurrected bundle property is added from a "default" that's independent of any template (.  I'll need to disable that for PyStein projects.